src/constants/base-contracts.ts

```diff
@@ -0,0 +1,96 @@
+const baseContracts = {
+  ERC721Base: `// SPDX-License-identifier: MIT

```

Comment:
These aren't actually base contracts - they're all `ERC721Base` which is the base contract used in all of them + extensions, so wonder if calling them all base contracts is inefficient. 

Also there's gotta be a better place to put these than in this repo, this is going to get huge as we add more extensions and will be tough to maintain.

Probably the best place this can come from is if we have docs for the the starter template code for all these bases somewhere, and then we can pull from there so it's easier to maintain.

Reply:
Yea agreed we should pull from github somewhere